python/lsst/ap/association/association.py

```diff
@@ -114,30 +114,39 @@ def run(self,
               not matched a new DiaSource. (`int`)
         """
         diaSources = self.check_dia_source_radec(diaSources)
+
+        if self.config.doTrailedSourceFilter:
+            diaTrailedResult = self.trailedSourceFilter.run(diaSources,
+                                                            exposure_time)
+            diaSources = diaTrailedResult.diaSources
+            trailedSources = diaTrailedResult.trailedDiaSources
+
+            self.log.info(
+                "%i DIASources exceed max_trail_length, dropping fromsource catalog." % len(
```

Comment:
```suggestion
                "%I Found one or more DiaSource that exceeds max_trail_length, dropping from source catalog." % len(
```
It might be even more informative if "num_filtered" were part of this log message.

Reply:
num_filtered is the same thing as the log message, they are both len(diaTrailedResult.trailedDiaSources). do you mean I should do something like `from source catalog. num_filtered: " % len( `

Reply:
OH sorry it's a super long line isn't it, hello that's what the `%i` is, my bad!